kitchen-tests/Gemfile.lock

```diff
@@ -59,9 +59,8 @@ GEM
     excon (0.51.0)
     faraday (0.9.2)
       multipart-post (>= 1.2, < 3)
-    ffi (1.9.14)
```

Comment:
i noticed that. I actually ran `rake dependencies:update` on ubuntu. Was gonna just let it be unless it breaks.